pmd-core/src/main/resources/rulesets/releases/6290.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
```

Comment:
I was not sure if I was supposed to create this file or not. Let me know if I should remove it from the PR.

Reply:
That's great, please leave it, thanks! We usually create this file when we merge the PR and update release notes.